File: backend/DataAccess/DaoGenerated/システム共有/tm_aftaskscheduleDto.cs
// *******************************************************************
// 業務名称　: 健康管理システム
// 機能概要　: DTO定義
//             タスクスケジュールマスタ
// 作成日　　: 2023.01.23
// 作成者　　: 屠
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.DataAccess
{
    public class tm_aftaskscheduleDto : DaEntityModelBase
    {
		public const string TABLE_NAME = "tm_aftaskschedule";
		public string taskid { get; set; }                        //タスクID
		public string tasknm { get; set; }                        //タスク名
		public string? renkeiid { get; set; }                     //HangFire連携ID
		public string? biko { get; set; }                         //説明
		public DateTime? zenjikkodttmf { get; set; }              //前回の実行日時（開始）
		public DateTime? zenjikkodttmt { get; set; }              //前回の実行日時（終了）
		public DateTime? jikaijikkodttmt { get; set; }            //次回実行日時
		public string syorikbn { get; set; }                      //処理区分
		public string gyomukbn { get; set; }                      //業務区分
		public string modulecd { get; set; }                      //モジュールコード
		public string? hikisu { get; set; }                       //引数
		public string yukoymdf { get; set; }                      //有効年月日（開始）
		public string yukotmf { get; set; }                       //有効時間（開始）
		public string hindokbn { get; set; }                      //頻度区分
		public bool syukustopflg { get; set; }                    //祝日停止フラグ
		public string? yobi { get; set; }                         //曜日
		public string? maitukihiyobikbn { get; set; }             //毎月の日・曜日区分
		public string? maitukituki { get; set; }                  //毎月の月
		public string? maitukihi { get; set; }                    //毎月の日
		public string? maitukisyu { get; set; }                   //毎月の週
		public string? statuscd { get; set; }                     //処理結果コード
		public bool? kurikaesikanflg { get; set; }                //繰り返し間隔フラグ
		public string? kurikaesikankbn { get; set; }              //繰り返し間隔区分
		public string? jikantaif { get; set; }                    //時間帯開始_時
		public string? jikantait { get; set; }                    //時間帯終了_時
		public string? jikannaif { get; set; }                    //時間内開始_分
		public string? jikannait { get; set; }                    //時間内終了_分
		public bool stopflg { get; set; }                         //使用停止フラグ
		public string reguserid { get; set; }                     //登録ユーザーID
		public DateTime regdttm { get; set; }                     //登録日時
		public string upduserid { get; set; }                     //更新ユーザーID
		public DateTime upddttm { get; set; }                     //更新日時
    }
}

File: GJSBackend/BusinessService/Db/DaoGenerated/システム共有/tm_aftaskscheduleDto.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: DTO定義
'             タスクスケジュールマスタ
' 作成日　　: 2024.07.23
' 作成者　　: 
' 変更履歴　:
' *******************************************************************
Namespace Jbd.Gjs.Db
    Public Class tm_aftaskscheduleDto
        Inherits DaEntityModelBase
        Public Const TABLE_NAME As String = "tm_aftaskschedule"
        Public Property taskid As String                        'タスクID
        Public Property tasknm As String                        'タスク名
        Public Property renkeiid As String                     'HangFire連携ID
        Public Property biko As String                         '説明
        Public Property zenjikkodttmf As Date              '前回の実行日時（開始）
        Public Property zenjikkodttmt As Date              '前回の実行日時（終了）
        Public Property jikaijikkodttmt As Date            '次回実行日時
        Public Property syorikbn As String                      '処理区分
        Public Property gyomukbn As String                      '業務区分
        Public Property modulecd As String                      'モジュールコード
        Public Property hikisu As String                       '引数
        Public Property yukoymdf As String                      '有効年月日（開始）
        Public Property yukotmf As String                       '有効時間（開始）
        Public Property hindokbn As String                      '頻度区分
        Public Property syukustopflg As Boolean                    '祝日停止フラグ
        Public Property yobi As String                         '曜日
        Public Property maitukihiyobikbn As String             '毎月の日・曜日区分
        Public Property maitukituki As String                  '毎月の月
        Public Property maitukihi As String                    '毎月の日
        Public Property maitukisyu As String                   '毎月の週
        Public Property statuscd As String                     '処理結果コード
        Public Property kurikaesikanflg As Boolean                '繰り返し間隔フラグ
        Public Property kurikaesikankbn As String              '繰り返し間隔区分
        Public Property jikantaif As String                    '時間帯開始_時
        Public Property jikantait As String                    '時間帯終了_時
        Public Property jikannaif As String                    '時間内開始_分
        Public Property jikannait As String                    '時間内終了_分
        Public Property stopflg As Boolean                         '使用停止フラグ
        Public Property reguserid As String                     '登録ユーザーID
        Public Property regdttm As Date                     '登録日時
        Public Property upduserid As String                     '更新ユーザーID
        Public Property upddttm As Date                     '更新日時
    End Class
End Namespace
